web/src/pages/profile.tsx

```diff
@@ -543,6 +550,16 @@ const ProfilePage = (props: Props): ReactElement => {
         >
           <HomeBasedBusiness />
         </ProfileField>
+
+        <ProfileField
+          fieldName="elevatorOwningBusiness"
+          displayAltDescription={displayAltHomeBasedBusinessDescription}
```

Comment:
dust - if we are displaying `displayAltHomeBasedBusinessDescription` why are we storing description  and altDescription?

Reply:
`displayAltHomeBasedBusinessDescription` determines whether or not the alt description is served